examples/docker/shiny-core/README.md

```diff
@@ -0,0 +1,58 @@
+# Model monitoring application
+
+This app is built on Shiny and allows you to visualize model training accuracy scores in realtime in 2 steps:
+
+* Train multiple models with different parameters locally, and store the accuracy scores to a remote database at the end of every epoch.
+* The visualisation application dynamically detects changes in the remote database and generates a plot as the training progresses.
+
+## Connecting to MongoDB Cloud
+
+For the purpose of this tutorial we will use the `MongoDB Atlas` cloud database. Let's see the steps for creating a cluster:
+
+First create an account on [MongoDB cloud](https://www.mongodb.com/atlas/database) and create a new project:
+
+![](./static/create-project.png)
+
+Then select `Create deployment` and select the free tier `M0` or your desired tier. 
+
+In the `Security Quickstart` page, select your authentication type. For this tutorial we have generated a username and password.
+
+Select the `Cloud Environment` and open the `Network Access Page` in a new tab.
+
+![](./static/network-access.png)
+
+Now select `ALLOW ACCESS FROM ANYWHERE` and confirm to allow all IP addresses to connect to the cluster.
+
+![](./static/ip-access.png)
+
+Once you have completed the access rules setup, select the `CONNECT` button on the `Overview` page to get the connection details. Specifically, copy the connection string and replace the `URI` variable value in `app-core.py`, `train_one.py` and `train_two.py`
```

Comment:
show how to set and environment variable and modify the scripts to read from there

Reply:
[fixed](https://github.com/ploomber/doc/pull/105/files#diff-883b5ca3546bed48155f3709e49ebe5d5b987079fdc8b9c9d9f930a943512616R42-R44)

[fixed](https://github.com/ploomber/doc/pull/105/files#diff-883b5ca3546bed48155f3709e49ebe5d5b987079fdc8b9c9d9f930a943512616R29)